utils_lib/src/cameligo/lib/rational.mligo

```diff
@@ -0,0 +1,36 @@
+#import "../lib/math.mligo" "Math"
+
+type rational = { p : int; q: int }
```

Comment:
should be just `t`
because on the user end: `Rational.rational` vs `Rational.t`